maven/bnd-export-maven-plugin/src/main/java/aQute/bnd/maven/export/plugin/ExportMojo.java

```diff
@@ -99,7 +103,6 @@ private void export(Run run) throws Exception {
 	private void resolve(StandaloneRun run) throws Exception, IOException {
 		try (ProjectResolver projectResolver = new ProjectResolver(run)) {
 			List<Container> runBundles = projectResolver.getRunBundles();
```

Comment:
it's done differently because Groovy can use a lambda and get the resolver instance. Can we do that without java 8 in this plugin?